includes/admin/class-sensei-editor-wizard.php

```diff
@@ -46,7 +46,8 @@ public static function instance() {
 	 */
 	public function init() {
 		add_action( 'init', [ $this, 'register_post_metas' ] );
-		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_scripts' ] );
+		// Priority 9 to make sure it will run before the block editor nux on WPCOM. While this code is written, it uses priority 100.
+		add_action( 'enqueue_block_editor_assets', [ $this, 'enqueue_editor_wizard_assets' ], 9 );
```

Comment:
The hook was changed to make sure it will be added before the block editor nux on WPCOM.